.circleci/export-e2e-test-user-vars.sh

```diff
@@ -19,20 +19,23 @@ do
        echo "export COLLABORATOR_USER=$PUPPETEER_COLLABORATOR_STAGING" >> "$BASH_ENV"
        echo "export USER_NAME=$PUPPETEER_USER_STAGING" >> "$BASH_ENV"
        echo "export ACCESS_TEST_USER=$PUPPETEER_ACCESS_STAGING" >> "$BASH_ENV"
+       echo "export ADMIN_TEST_USER=$PUPPETEER_ADMIN_STAGING" >> "$BASH_ENV"
```

Comment:
Just a heads up that there will be some conflicts between this PR and https://github.com/all-of-us/workbench/pull/5451 @aweng98

Reply:
In that case, I'll also consult @aweng98 before merging this branch. Thank you @calbach!